roles/provisioner/virsh/tasks/create_vms.yml

```diff
@@ -0,0 +1,78 @@
+---
+- name: set fact for node
+  set_fact:
+      node_data: "{{ item.value }}"
+      node_name: "{{ item.key }}"
+
+- name: import vm disk image(s) for {{ node_name }}
+  get_url:
+      url: "{{ item.value.import_url }}"
+      dest: /var/lib/libvirt/images/{{node_name}}_original_{{item.key}}.qcow2
+      timeout: 30
+  register: result
+  until: result.msg.find("Request failed") == -1
+  retries: 5
+  delay: 5
+  when: item.value.import_url is defined and item.value.import_url
+  with_dict: "{{ node_data.disks }}"
```

Comment:
If several node types have the same image, will this download the same file multiple times with different `dest` file names?